client/src/components/ElementEditor/Header.js

```diff
@@ -63,14 +71,22 @@ class Header extends Component {
           </div>
           <h3 className="element-editor-header__title">{title}</h3>
         </div>
-        <FormActionComponent
-          onClick={this.handleDelete}
-          title={null}
-          icon="trash-bin"
-          attributes={{
-            title: i18n._t('ElementHeader.DELETE', 'Delete'),
-          }}
-        />
+        <div className="element-editor-header__actions">
+          <ActionMenuComponent
+            id={this.props.id}
```

Comment:
Should we prefix this with something unique to the actions menu maybe like `element-editor-actions-${id}` or something? Note `id` is destructured already at this point

Reply:
Good point. Changed it as suggested.

Reply:
Also FYI - IDs that start with a number is invalid HTML!

Reply:
😮